### ISSUE TITLE
YAML file for 02691156_2048_pc

### ISSUE THREAD

**Issue by ehedlin** [Rgtemze/HyperDiffusion#8]
what was the configs/overfitting_configs/overfit_plane.yaml file used for the provided points? I cant seem to find settings that converge with the provided points. Were these the points used to train the provided weight files?

**Comment by ehedlin** [comment]
Also it seems the loss value goes to negative infinity and its likely because the function is not properly normalized as an occupancy grid https://github.com/Rgtemze/HyperDiffusion/blob/fd2c5812aa62bc278061ccbcdd0185ca9ce46ed0/mlp_models.py#L98
